practices/bad_team_onboarding.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: agile-events
+stages:
+  - development
+short_description: The bad team onboarding is when your new members don't receive the crucial information or vice-versa i.e. they are overflooded with the pointers and information at the pace.
+tags:
+  - retrospective
+  - refinement
+  - demotivated team
+  - less productivity
+keywords:
+  - demotivated team
+  - retrospective
+  - meaningless work
+---
+
+# Bad Team Onboarding
+
+**TL;DR**
+
+A great onboarding process should help an employee identify with the company and feel a sense of belonging–like joining a family that welcomes you with open arms, but when this feeling is lost somewhere it leds to Bad Team Onboarding
+
+
+## What Is Bad Team Onboarding
+
+Bad Team Onboarding refers to ruining entire experience of hiring, welcoming, orienting, and engaging a new hire and helping them become acclimated and integrated into the corporate culture.
+![Onboarding](/files/bad_team_onboarding.jpg)
+
+## Why Bad Team Onboarding Occured
+1. NO PLAN, NO OBJECTIVES
+2. IT MOVES TOO FAST
+3. INADEQUATE NEW HIRE TRAINING
+4. NO FOLLOW UP, SUPPORT OR PERFORMANCE MEASUREMENT
+
+
+## How to Avoid Bad Team Onboarding
+- Provide your new hire with a transparent and open explanation of the organizational structure of the organization. 
+- Talk about opportunities that your team members have and that have been provided in the past.
+- Have an open one-on-one meeting with your new team member. 
+- Employees do not need to spend time in a room filling out paperwork for their benefits enrollment, their tax forms, etc.
+- Give a full tour so that a new hire knows where everything is. Sometimes, comfort level is heightened beyond what we know when we spend time on the minor things that we take for granted.
+- All team members must be welcoming, friendly, and genuinely happy to have this new member on board. A special lunch or happy hour is called for.
```

Comment:
Please, consider rewriting this since it seems to similar to _All team members must be welcoming, friendly, and genuinely happy to have this new member on board. A special lunch or happy hour is called for_ from https://talentculture.com/bad-hire-or-bad-onboarding-10-tips-for-successful-new-hire-assimilation/